src/app/nextversion/nextversion.go

```diff
@@ -154,6 +162,14 @@ func NextVersion(cCtx *cli.Context) error {
 
 	case nextOverride == nil && next != nil:
 		nextStr = next.String()
+		// If we don't have an override, the bumper did not bump anything, and the user has set `--fail`, then we should error out.
+		if errors.Is(err, bumper.ErrNoNewVersion) {
+			log.Warnf("None of the changelog entries produced a version bump, returning current version")
+			a := cCtx.Bool("fail")
+			if a {
```

Comment:
This `a` looks like a debugging leftover 😉

Reply:
yes...

Reply:
fixed